Packs/IntSight/Integrations/Rapid7ThreatCommand/Rapid7ThreatCommand.yml

```
@@ -0,0 +1,2433 @@
+  - BlackMarkets
+  - HackingForums
+  - SocialMedia
+  - PasteSites
+  - Others
+  required: false
+  type: 16
+- defaultvalue: 'false'
+  display: Fetch closed alerts
+  name: fetch_closed_incidents
+  required: false
+  type: 8
+- defaultvalue: 'false'
+  display: Include CSV files of alerts
+  name: fetch_csv
+  required: false
+  type: 8
+- defaultvalue: 'false'
+  display: Include attachments of alerts
+  name: fetch_attachments
+  required: false
+  type: 8
+  additionalinfo: MSSP accounts must provide a sub-account ID to perform this action.
+- display: "Sub-account ID (for MSSP accounts)."
+  name: mssp_sub_account
+  type: 0
+  required: false
+- display: Incident type
+  name: incidentType
+  required: false
+  type: 13
+- defaultvalue: 'false'
+  display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- defaultvalue: 'false'
+  display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+description: Rapid7 Insight - Threat Command allows managing alerts, CVEs, IOCs, and assets by accounts and MSSP accounts.
+display: Rapid7 - Threat Command (IntSights)
+name: rapid7_threat_command
+script:
+  commands:
+  - arguments:
+    - description: Filter by free text, which can be the cyber term name or ID.
+      isArray: false
+      name: search
+      required: false
+    - auto: PREDEFINED
+      description: "Filter by one or more cyber term types. For example: ThreatActor,Campaign."
+      isArray: true
+      name: types
+      predefined:
+      - Threat Actor
+      - Malware
+      - Campaign
+      required: false
+    - auto: PREDEFINED
+      description: "Filter by one or more cyber term severities. For example: High,Low."
+      isArray: false
+      name: severities
+      predefined:
+      - High
+      - Medium
+      - Low
+      required: false
+    - description: "Filter by one or more targeted sectors. For example: Education,Government."
+      isArray: true
+      name: sectors
+      required: false
+    - description: "Filter by one or more targeted countries. For example: Albania,Algeria."
+      isArray: true
+      name: countries
+      required: false
+    - description: "Filter by one or more nationalities. For example: Egypt,Iraq."
+      isArray: true
+      name: origins
+      required: false
+    - description: "Filter by one or more TTPs. For example: Malware,Backdoor"
+      isArray: true
+      name: ttps
+      required: false
+    - description: "Filter by last update date is greater than the given value (in ISO 8601 format). For example:  2022-12-25T08:38:06Z. Default value: Last year."
+      isArray: false
+      name: last_update_from
+      required: false
+    - description: "Filter by last update date is less than the given value (in ISO 8601 format). For example:  2022-12-25T08:38:06Z."
+      isArray: false
+      name: last_update_to
+      required: false
+    - description: The page number of the results to retrieve.
```

I understand page argument is 1-based, worth specifying.
```suggestion
    - defaultValue: '1'
      description: The page number of the results to retrieve (1-based).
```